docs/topics/native/native-command-line-compiler.md

```diff
@@ -36,8 +40,8 @@ kotlinc-native hello.kt -o hello
 
 The value of `-o` option specifies the name of the output file, so this call should generate a `hello.kexe` (Linux and macOS)
 or `hello.exe` (Windows) binary file.
-For the full list of available compiler options, see the [compiler options reference](compiler-reference.md).
+For the full list of available options, see [Kotlin compiler options](compiler-reference.md).
 
```

Comment:
Reading this, I would expect the outcome here, as in "and now let's run the program we compiled." But that's your call as to how necessary that is.

Reply:
Added a separation section below